Examples/Framework/ML/src/NeuralCalibrator.cpp

```diff
@@ -96,8 +97,8 @@ void ActsExamples::NeuralCalibrator::calibrate(
 
   input[iInput++] = idxSourceLink.geometryId().volume();
   input[iInput++] = idxSourceLink.geometryId().layer();
-  input[iInput++] = trackState.parameters()[Acts::eBoundPhi];
-  input[iInput++] = trackState.parameters()[Acts::eBoundTheta];
+
+  const Acts::Surface& surf = trackState.referenceSurface();
```

Comment:
```suggestion
  const Acts::Surface& referenceSurface = trackState.referenceSurface();
```
nitpick: makes the code easier to read (at least for me)

Reply:
Done